utils/converter/src/main/java/io/apicurio/registry/utils/converter/avro/AvroData.java

```diff
@@ -1487,7 +1488,7 @@ private Object toConnectData(Schema schema, Object value, ToConnectContext toCon
                     throw new DataException("Unknown Connect schema type: " + schema.type());
             }
 
-            if (schema != null && schema.name() != null && doLogicalConversion) {
+            if (schema.name() != null && doLogicalConversion) {
```

Comment:
Why removing the null check from the schema name?

Reply:
Based on the IDE analysis, the condition always appears to be true. To reduce the delta, I will add the null check back.